recipes/libuvc/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake ${CMAKE_ARGS} .. \
+      -G "Ninja" \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX -DCMAKE_INSTALL_PREFIX=$PREFIX \
```

Comment:
```suggestion
      -DCMAKE_PREFIX_PATH=$PREFIX \
      -DCMAKE_INSTALL_PREFIX=$PREFIX \
```

Reply:
Rather delete the whole line as these arguments are all part of CMSKE_ARGS.

Reply:
I am confused - which line(s) should go away?